src/stream/chacha/chacha_setup.c

```diff
@@ -16,8 +16,8 @@
 
 #ifdef LTC_CHACHA
 
-static const char sigma[16] = "expand 32-byte k";
-static const char tau[16]   = "expand 16-byte k";
+static const char * const sigma = "expand 32-byte k";
+static const char * const tau   = "expand 16-byte k";
```

Comment:
I like it as proposed as it's the most natural & strict way to declare these pointers.